### PR TITLE
feat: Issue #62 - Add simple support for const

### PR DESCRIPTION
**Pull request by tlgman**
- Add support for "const" field
- Add comments to explain that support is not total

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -114,6 +114,66 @@ describe('Simple schemas', () => {
...
+ * @typedef {23.3}
+ */
+`
+    expect(generate(schema)).toEqual(expected)
+  })
+
+  it('Integer with const', function () {
+    const schema = {
+      type: 'integer',
+      const: '42'
```
Comment (brettz9) [inline test.js:144]: Should this be `const: 42` instead? Likewise should the two `const`  below also be changed from a string to the value in question?

Comment (tlgman) [reply test.js]: You're absolutely right.
I changed it

**Comment by brettz9** [comment]
LGTM! @n3ps : Did you want to do your own review?

**Comment by n3ps** [comment]
Thanks @tlgman for adding this and @brettz9 for the review. Checked it and lgtm.

**Pull request merged**